demo/client/src/3d-demo.ts

```diff
@@ -376,6 +376,7 @@ class ThreeDRenderer extends Renderer {
     camera: PerspectiveCamera;
 
     needsStencil: boolean = false;
+    rightEye: boolean = false;
```

Comment:
nit: put a newline above this. Also maybe call this `vrRightEye` to make it clear that this is for VR?